virttest/utils_net.py

```diff
@@ -1894,7 +1894,10 @@ def ovs_br_exists(brname, ovs=None):
     if ovs is None:
         ovs = __ovs
 
-    return brname in ovs.list_br()
+    if ovs is not None:
```

Comment:
return false and loging warning looks better if not a OVS system.

Reply:
@xutian Given so, I'm afraid it will hard to distinguish the False is due to brname not exist or OVS not support. And raise exception here shall have no impact on non ovs functions, since this func is dedicated to ovs check.

Reply:
Agree with @hahakiki2010

Reply:
we should consider using different way to check whether ovs is supported or not instead of using ovs_br_exists(). This API is designed to return None if ovs doesn't exist.

Reply:
@balamuruhans, agreed with you to return False here, to keep this API dedicated to ovs bridge check
Then what if we raise ovs not support exception in add_ovs_bridge, because this func has the same issue as ovs_br_exists. When libvirt test case iface_ovs.py detect ovs bridge not exist, and try to add_ovs_bridge, exception like  "NoneType' object has no attribute 'add_br'" will be raised too.